docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+for SECRET in LLDAP_JWT_SECRET LLDAP_LDAP_USER_PASS; do
+    FILE_VAR="${SECRET}_FILE"
+    SECRET_FILE="${!FILE_VAR:-}"
```

Comment:
in sh, you can replace that with
SECRET_FILE="$(eval echo \$$FILE_VAR)"

if I'm not mistaken